Makefile

```diff
@@ -152,3 +153,9 @@ infoblox-secret:
 	kubectl -n ohmyglb create secret generic external-dns \
 	    --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_USERNAME=$${WAPI_USERNAME} \
 	    --from-literal=EXTERNAL_DNS_INFOBLOX_WAPI_PASSWORD=$${WAPI_PASSWORD}
+
+.PHONY: test-round-robin
+test-round-robin:
+	@kubectl run -it --rm busybox --restart=Never --image=busybox -- sh -c \
+		"echo 'nameserver `$(OHMYGLB_COREDNS_IP)`' > /etc/resolv.conf && \
```

Comment:
why backtics are required here >

Reply:
@ytsarev  because `OHMYGLB_COREDNS_IP` is not environment variable but GNU make variable which is kubectl command  

`kubectl get svc ohmyglb-coredns -n ohmyglb -o custom-columns='IP:spec.clusterIP' --no-headers`

with backtics it  will process like:
```
@kubectl run -it --rm busybox --restart=Never --image=busybox -- sh -c \
		"echo 'nameserver `kubectl get svc ohmyglb-coredns -n ohmyglb -o custom-columns='IP:spec.clusterIP' --no-headers`' > /etc/resolv.conf &&
```